addons/binding/org.openhab.binding.modbus/README.md

```diff
@@ -763,6 +763,76 @@ sitemap modbus_ex_scaling label="modbus_ex_scaling"
 
 See [transformation example](#transformation-example-scaling) for the `divide10.js` and `multiply10.js`.
 
+### Dimmer Example
+
+_(thanks to [rossko57](https://community.openhab.org/t/modbus-openhab2-binding-available-for-alpha-testing/27657/560?u=ssalonen)!)_
+
+OpenHAB Dimmer type Items are not a straightforward match to Modbus registers, as they feature a numeric value which is limited to 0-100 Percent, as well as handling ON/OFF commands.
```

Comment:
I'd remove OpenHAB here so the docs make more sense if they are one day used in another ESH derivative. It would also fix the issue that it should be spelled as "openHAB". :wink:

Reply:
Fixed occurences of "openhab", removed the openHAB reference in this sentence